fre/pp/ppyaml_test/pp_c96L65.yaml

```diff
@@ -21,10 +21,10 @@ rose-suite:
     do_analysis: False
 
   directories:
-    history_dir: "/archive/c2b/am5/2022.01/$(name)/$(platform)-$(target)/history"
-    pp_dir: "/archive/$USER/canopy/am5/2022.01/$(name)/$(platform)-$(target)/pp" 
+    history_dir: "/archive/c2b/$(stem)/$(name)/$(platform)-$(target)/history"
+    pp_dir: "/archive/$USER/canopy/$(stem)/$(name)/$(platform)-$(target)/pp" 
     ptmp_dir: "/xtmp/$USER/ptmp"
-    analysis: "/nbhome/$USER/canopy/analysis/$(name)" 
+    analysis: "/nbhome/$USER/canopy/analysis/$(name)"
```

Comment:
I have a strong feeling that `directories` should be set above the experiments, along with the FRE properties in the "main" yaml.

There's nothing in these definitions that are really experiment-specific except the experiment name itself.

Reply:
Aren't the directories experiment specific though with the name? I feel like it makes sense to keep them in the experiment's ppyaml because they would be paths for that experiment.

Reply:
Or at least putting them under the associated `expname` field

Reply:
The best argument for keeping it in the main yaml is minimizing duplication. Almost certainly, all the directories will be the same, except for the experiment name, stem, platform, and target.

Another argument is similarity to how Bronx handles directories. In Bronx, the directories are set per-platform, not per-experiment.